g2core/stepper.h

```diff
@@ -430,15 +431,17 @@ extern stPrepSingleton_t st_pre;            // only used by config_app diagnosti
 /**** Stepper (base object) ****/
 
 struct Stepper {
+protected:
     Timeout _motor_disable_timeout;         // this is the timeout object that will let us know when time is u
     uint32_t _motor_disable_timeout_ms;     // the number of ms that the timeout is reset to
+    ioMode _motor_enable_polarity;         // 0=active HIGH, 1=active LOW
     stPowerState _power_state;              // state machine for managing motor power
     stPowerMode _power_mode;                // See stPowerMode for values
 
     /* stepper default values */
-
+public:
     // sets default pwm freq for all motor vrefs (commented line below also sets HiZ)
```

Comment:
Presumably this comment should also go, since `frequency` is deleted below?

Reply:
Good catch, yes, this should go as well.